site/src/modules/dashboard/Navbar/DeploymentDropdown.tsx

```diff
@@ -13,22 +13,25 @@ import {
 import { USERS_LINK } from "modules/navigation";
 
 interface DeploymentDropdownProps {
-  canViewAuditLog: boolean;
   canViewDeployment: boolean;
+  canViewOrganizations: boolean;
   canViewAllUsers: boolean;
+  canViewAuditLog: boolean;
   canViewHealth: boolean;
```

Comment:
Instead of passing all of these permissions using props, wdyt about getting them directly from `useDeployment`?

Reply:
This PR is already pretty big, so I'm going to say that's out of scope. 😅